torchgeo/datasets/idtrees.py

```diff
@@ -612,10 +613,10 @@ def plot_las(self, index: int, colormap: Optional[str] = None) -> Any:
             else:
                 colors = np.zeros_like(points)
 
-        pcd = open3d.geometry.PointCloud()
-        pcd.points = open3d.utility.Vector3dVector(points)
-        pcd.colors = open3d.utility.Vector3dVector(colors)
-        vis = open3d.visualization.Visualizer()
-        vis.create_window()
-        vis.add_geometry(pcd)
-        return vis
+        point_cloud = pyvista.PolyData(points)  # type: ignore[attr-defined]
+        point_cloud["color"] = colors
+        plotter = pyvista.Plotter()  # type: ignore[attr-defined]
+        _ = plotter.add_mesh(point_cloud, scalars="color", rgb=True)
```

Comment:
I recommend changing this section to PyVista's `plot()` API instead. That way, you can easily capture user-passed `kwargs` and pass them to PyVista to let users easily change the display parameters (e.g., `point_size`) and/or change Jupyter plotting backends, rendering techniques like eye-dome lighting, etc.

This would look like:

```py
return point_cloud.plot(scalars="color", rgb=True, **kwargs)
```